text/text.go

```diff
@@ -189,6 +190,7 @@ func (txt *Text) Clear() {
 	txt.bounds = pixel.Rect{}
 	txt.tris.SetLen(0)
 	txt.dirty = true
+	txt.Dot = txt.Orig
```

Comment:
Needs to be mentioned in the documentation of the method. Also, the Text type documentation says that to reset Dot, you asign Orig to it. This line is redundant now and can be removed.

Reply:
Yes, I'll fix this right away.